http-verbs-common/src/main/scala/uk/gov/hmrc/play/http/ws/WSHttpResponse.scala

```diff
@@ -29,3 +33,16 @@ class WSHttpResponse(wsResponse: WSResponse) extends HttpResponse {
 
   override def body: String = wsResponse.body
 }
+
+object WSHttpResponse {
+  @silent("deprecated") // allHeaders is required for Play 2.5
```

Comment:
Isn't this using wsResponse.allHeaders which isn't deprecated? (HttpResponse.allHeaders is deprecated)

Reply:
I could be wrong

Reply:
This silence is for `WsResponse.allHeaders` which has been deprecated for Play 2.6+ (we still need to use it for Play 2.5 - I've just silenced the warning, rather than creating a different implementations for each play version).
Our `HttpResponse.allHeaders` has also been deprecated.